src/vivarium/framework/population/manager.py

```diff
@@ -222,18 +222,17 @@ def get_view(
         ----------
         columns
             A subset of the state table columns that will be available in the
-            returned view. If empty, this view will have access to the entire
-            state table.
+            returned view. If requires_all_columns is True, this will be the
```

Comment:
Nit: can you make this wording change everywhere we say this? I prefer that, because setting the this argument doesn't actually change what columns the component will create - we're just telling the user what they should put here. 
```suggestion
            returned view. If requires_all_columns is True, this should be set to the
```